storops/unity/client.py

```diff
@@ -33,6 +35,18 @@
 log = logging.getLogger(__name__)
 
 
+def wrap_not_supported(func):
+    @wraps(func)
+    def _wrap(*args, **kwargs):
+        try:
+            ret = func(*args, **kwargs)
+        except UnityResourceNotSupportedError:
+            log.info('Resouce type [{}] is not supported.'.format(args[0]))
```

Comment:
Not suggest to add here.  First, `args[0]` is not intuitive.  And you cannot assure this function is only used for missing type.  At least I cannot see this from the wrapper name.  It's better to add it where you raise the exception.